src/incidencematrix.jl

```diff
@@ -50,6 +50,20 @@ function IncidenceMatrix{Symmetric}(mat::AbstractSparseMatrix)
     return res
 end
 
+function IncidenceMatrix{NonSymmetric}(TrueIndices::Base.Array{Base.Array{Int64,1},1}) where T
+    m = length(TrueIndices)
+    n = maximum([maximum(set) for set in TrueIndices])
```

Comment:
this allocates a vector only to compute its maximum. maybe 
```suggestion
    n = maximum(maximum, TrueIndices)
```
would work?

Reply:
also I'm not sure if is `Base.Vector{Base.Vector}` strictly necessary here? at `AbstractVector{<:AbstractVector{<:Integer}}` should be just fine?